src/lang/english.txt

```diff
@@ -5128,3 +5130,36 @@ STR_PLANE                                                       :{BLACK}{PLANE}
 STR_SHIP                                                        :{BLACK}{SHIP}
 
 STR_TOOLBAR_RAILTYPE_VELOCITY                                   :{STRING} ({VELOCITY})
+
+STR_BUILD_VEHICLE_SMART                                         :Smart build vehicle menu
+STR_BUILD_VEHICLE_SMART_HELPTEXT                                :Smart build vehicle menu
+STR_BUILD_VEHICLE_DETAIL                                        :{BLACK}Detail
+STR_BUILD_VEHICLE_DETAIL_TOOLTIP                                :{BLACK}Extended options for comfortable unit select
+STR_BUILD_VEHICLE_AUTO_TRAIN                                    :{BLACK}Take the train parameters from the from
+STR_BUILD_VEHICLE_AUTO_TRAIN_TOOLTIP                            :{BLACK}Take parameters from the first train from the depot (maximum weight, available speed, train length)
+STR_BUILD_VEHICLE_FILTER                                        :{BLACK}Use filters...
+STR_BUILD_VEHICLE_FILTER_TOOLTIP                                :{BLACK}Open filters panel
+STR_BUILD_VEHICLE_INCLINE_LABEL                                 :{BLACK}Allow slopes:
+STR_BUILD_VEHICLE_INCLINE_EDITOR                                :{ORANGE}{NUM}%
+STR_BUILD_VEHICLE_INCLINE_TOOLTIP                               :{BLACK}An opportunity to overcome the incline
+STR_LOCOMOTIVES                                                 :{BLACK}Locomotives
+STR_MOTOR_WAGONS                                                :{BLACK}Motor-Wagons
```

Comment:
The game normally doesn't refer to power cars specifically, but if it were to, "motor wagons" would not be the term.

Also, this string-ID is too generic, it still needs the context of which window it belongs to.